cli/cmd/lib_cluster_config.go

```diff
@@ -343,6 +343,66 @@ func setConfigFieldsFromCached(userClusterConfig *clusterconfig.Config, cachedCl
 	}
 	userClusterConfig.APIGatewaySetting = cachedClusterConfig.APIGatewaySetting
 
+	if s.Obj(cachedClusterConfig.VPCCIDR) != s.Obj(userClusterConfig.VPCCIDR) {
+		return clusterconfig.ErrorConfigCannotBeChangedOnUpdate(clusterconfig.VPCCIDRKey, cachedClusterConfig.VPCCIDR)
+	}
+	userClusterConfig.VPCCIDR = cachedClusterConfig.VPCCIDR
+
+	if s.Obj(cachedClusterConfig.ImageDownloader) != s.Obj(userClusterConfig.ImageDownloader) {
+		return clusterconfig.ErrorConfigCannotBeChangedOnUpdate(clusterconfig.ImageDownloaderKey, cachedClusterConfig.ImageDownloader)
+	}
+	userClusterConfig.ImageDownloader = cachedClusterConfig.ImageDownloader
+
+	if s.Obj(cachedClusterConfig.ImageRequestMonitor) != s.Obj(userClusterConfig.ImageRequestMonitor) {
+		return clusterconfig.ErrorConfigCannotBeChangedOnUpdate(clusterconfig.ImageRequestMonitorKey, cachedClusterConfig.ImageRequestMonitor)
+	}
+	userClusterConfig.ImageDownloader = cachedClusterConfig.ImageDownloader
```

Comment:
This should be `userClusterConfig. ImageRequestMonitor = cachedClusterConfig. ImageRequestMonitor `. There are several occurrences of this below.

Reply:
Good catch! sorry for the copy-paste error